markets/perps-market/contracts/storage/PerpsAccount.sol

```diff
@@ -400,19 +400,19 @@ library PerpsAccount {
                     continue;
                 }
 
-                runtime.amountToLiquidatePercentage = amountToLiquidate.divDecimal(
+                runtime.amountToLiquidateRatioD18 = amountToLiquidate.divDecimal(
                     MathUtil.abs(oldPositionSize)
                 );
 
-                runtime.percentageOfTotalLosingPnl = MathUtil.abs(totalPnl).divDecimal(
+                runtime.totalLosingPnlRatioD18 = MathUtil.abs(totalPnl).divDecimal(
                     runtime.totalLosingPnl
                 );
                 runtime.totalAvailableForDeposit = totalAvailableUsd.mulDecimal(
-                    runtime.percentageOfTotalLosingPnl
+                    runtime.totalLosingPnlRatioD18
                 );
 
                 runtime.amountToDeposit = runtime.totalAvailableForDeposit.mulDecimal(
-                    runtime.amountToLiquidatePercentage
+                    runtime.amountToLiquidateRatioD18
```

Comment:
ignore liquidation ones, they are all about to change.  ty

Reply:
if they gonna change than I can leave it anyway?